package.json

```diff
@@ -15,11 +22,19 @@
     "react-dom": "17.0.2"
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.15.1",
+    "@testing-library/react": "^12.1.2",
```

Comment:
Might want to look at adding [`@testing-library/cypress`](https://testing-library.com/docs/cypress-testing-library/intro/)

Reply:
I've added it as a dependency; I don't know if its configured correctly, since I don't use it any of the tests